internal/ddt/node_doc.go

```diff
@@ -122,6 +123,88 @@ func (d NodeDoc) Components() ([]*NodeDocComponent, error) {
 	return components, nil
 }
 
+// A headline used in the ToC
```

Comment:
How do you usually handle abreviations? I try to introduce them once, or try to stick to the long form if it doesn't get too annoying. Abbreviations in code is something different.

Reply:
Good point, fixed it